<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weatherclass</class>
 <widget class="QWidget" name="weatherclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="pm10Value24">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>254</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>미세먼지 농도</string>
   </property>
  </widget>
  <widget class="QLabel" name="pm10grade">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pm10gradetxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>미세먼지 상태</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
